src/rules/dollar-variable-empty-line-before/__tests__/index.js

```diff
@@ -334,6 +367,11 @@ testRule(rule, {
     {
       code: `a {
       width: 1; $var2: 2;
+    }`,
+      fixed: `a {
+      width: 1;
+
+$var2: 2;
```

Comment:
This is again a bit problematic, the variable is not indented when a newline is added.

Reply:
See my [comment](https://github.com/kristerkari/stylelint-scss/pull/231#discussion_r173961660) regarding the indentation problem in autofixing - we should take a similar approach here.

Reply:
@OriR I have been working on other stuff the for the last week or so, but I have now added `disableFix` option to this rule.

Would be nice to get the fixes in and release a new version :)

Reply:
Yeah, I didn't have much time going over the PRs as well.

I'll try and implement it as soon as possible 👍

Reply:
Don't worry, there's no hurry :) I'm just anxious with this stuff

Reply:
So the `disableFix` should appear only here and `dollar-variable-colon-newline-after` since they're the "debatable" rules?
Is there any other rule?

Reply:
I think that it's only those 2 rules with indentation issues.